src/Cowsay.zig

```diff
@@ -142,30 +122,27 @@ pub fn useCowFile(self: *Self, filename: []const u8) void {
     const file = std.fs.cwd().openFile(filename, .{}) catch |err| {
         // use default
         self.useDefaultCow();
-        _ = err catch {};
+        std.log.err("File \"{s}\": {s}", .{ filename, @errorName(err) });
```

Comment:
Errors can be logged like this, or `switch`'ed on to handle different errors separately. Otherwise, if you want to just ignore the error, you do not have to capture the error at all; just use `.openFile(filename, .{}) catch {`